tests/test_cache.py

```diff
@@ -146,3 +143,41 @@ def test_set(self):
 
             backend.set(res, res.value)
             assert dj_cache_set.called
+
+
+class TestFlaskCacheBackend(TestBaseCacheBackend):
+
+    def get_backend(self, **kwargs):
+        flask_cache = pytest.importorskip("nap.cache.flask_cache")
+        defaults = {
+            'default_timeout': DEFAULT_TIMEOUT,
+            'obey_cache_headers': True,
+            'config': {
+                'CACHE_TYPE': 'simple',
+                'CACHE_KEY_PREFIX': '',
+            }
+        }
+        from flask import Flask
+        app = Flask(__name__, static_url_path='/store/static')
+        defaults.update(kwargs)
+        return flask_cache.FlaskCacheBackend(app, **defaults)
+
+    def test_get(self):
+        backend = self.get_backend()
+        with mock.patch('flask_caching.Cache.get') as fl_cache_get:
```

Comment:
You could re-org the context manager, `res` Mock setup, and assertion so that only the call to `backend.get(res)` is managed within the `mock.patch` block, potentially making it more clear what patch is needed to have the test pass.  (same applies to test below, too).

Reply:
Good point, I like the idea of minimizing the scope of mocks